scrapers/infernalrestraints.yml

```diff
@@ -0,0 +1,40 @@
+name: infernalrestraints
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - infernalrestraints.com/device
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //a[@class="articleTitleText"][1]/text()
+      Date:
+        selector: //span[@class="articlePostDateText"][1]/text()
+        postProcess:
+          - parseDate: "January 02, 2006"
+          - parseDate: "January 2, 2006"
+      Details:
+        selector: //td[@class="articleCopyText"]/text()
+        concat: "<br>"
+        postProcess:
+          - replace:
+            - regex: <br>
+              with: "\n"
+      Performers:
+        Name: //a[@class="articleTitleText"][2]/text()
+      Tags:
+        Name:
+          selector: //td[contains(text(),"Tags:")]/text()
+          # To remove the Tags: text from the tags
+          postProcess:
+            - replace:
+              - regex: "Tags:"
+                with:
+          split: ","
+      Studio:
+        Name:
+            fixed: Infernal Restraints
+      Image: //*[@id="my-video"]/@poster
+      URL: //meta[@name='og:url']/@content
```

Comment:
is the og:url meta present in any scenes ? couldn't find a sample with it.

Reply:
forgot to remove, inherited from another scrapper.